scripts/validate-code.sh

```diff
@@ -2,4 +2,4 @@
 
 # Copyright (C) Lightbend Inc. <https://www.lightbend.com>
 
-sbt ";+validateCode;+mimaReportBinaryIssues" || exit 1
+sbt "validateCode;+mimaReportBinaryIssues" || exit 1
```

Comment:
the cross build for `validateCode` was the issue. This is a command alias. 

It was emitting

> [warn] Issuing a cross building command, but not all sub projects have the same cross build configuration. This could result in subprojects cross building against Scala versions that they are not compatible with. Try issuing cross building command with tasks instead, since sbt will be able to ensure that cross building is only done using configured project and Scala version combinations that are configured.

and then ignoring the code. We haven't done any formatting for a while in this project.